src/extras/SceneUtils.js

```diff
@@ -37,9 +41,200 @@ var SceneUtils = {
 		scene.remove( child );
 		parent.add( child );
 
+	},
+
+	convertFromZUp: function ( root, animations ) {
+
+		// see https://gamedev.stackexchange.com/a/7932
```

Comment:
I would not consider that as an authoritative link, and would remove the reference.

`matrix` is a change-of-basis matrix, and this method performs a change-of basis from...